operations/host_spawn.go

```diff
@@ -126,6 +132,7 @@ func hostCreate() cli.Command {
 				KeyName:      key,
 				UserData:     script,
 				InstanceTags: tags,
+				InstanceType: instanceType,
```

Comment:
Can we add validation (here or in Create/Modify) that the instance type is one we support? I imagine we have a list because when you spawn from the Spawn Host page you get a dropdown menu, but I'm not sure where it is.

Reply:
I think the dropdown list on the spawn host page is just distros? Don't think a list of allowed instance types exists (right now it's entirely determined by the provider settings embedded in each distro document) but definitely a good idea to have some kind of checks.